lib/team.rb

```diff
@@ -0,0 +1,28 @@
+require './lib/player'
+
+class Team
+  attr_reader :country,
+              :players
+
+  attr_writer :eliminated
```

Comment:
Like that you went with an `attr_writer` here and not just automatically to an `attr_accessor`!